src/complete_hack.py

```diff
@@ -36,6 +36,9 @@ def _check_pending_node(self):
         if now > expires:
             self._pending_node['status'] = "completed"
             self._db.submit({'node': self._pending_node})
+            checkout_node = self._db.get_node(self._pending_node['parent'])
+            checkout_node['status'] = "completed"
+            self._db.submit({'node': checkout_node})
```

Comment:
I don't think this is needed here anymore.
https://github.com/kernelci/kernelci-pipeline/pull/66 will take care of setting the node status to 'complete'.

Reply:
The idea is to have this temporary service in place now and then replace it with the definitive one from #66 when it's ready.

Reply:
I think the easier way is to merge #66 first than merging this one and dropping some changes later. I guess #66 is ready to be merged.

Reply:
No, #66 is not working which is why I made this and also to get all the changes related to the email reports merged.